### PR TITLE
[receiver/postgresql] Fix unit field in metadata.yaml

### PR DESCRIPTION
**Pull request by dmitryax**

### CODE REVIEW

**Review by bogdandrutu — APPROVED** [review]

File: receiver/postgresqlreceiver/metadata.yaml [hunk 1]
```diff
@@ -19,49 +19,49 @@ attributes:
 metrics:
   postgresql.blocks_read:
     description: The number of blocks read.
-    units: 1
+    unit: 1
```
Comment (codeboten) [inline receiver/postgresqlreceiver/metadata.yaml:22]: The unit `1` is confusing to me, is this specified anywhere?

File: receiver/postgresqlreceiver/metadata.yaml [hunk 2]
```diff
@@ -19,49 +19,49 @@ attributes:
...
     sum:
       monotonic: true
       aggregation: cumulative
     attributes: [ database ]
   postgresql.db_size:
     description: The database disk usage.
-    units: By
+    unit: By
```
Comment (codeboten) [inline receiver/postgresqlreceiver/metadata.yaml:36]: Is the unit `By` standard? I would have expected either `B` or `bytes` for bytes.

Comment (dmitryax) [reply receiver/postgresqlreceiver/metadata.yaml:36]: AFAIK we follow codes from https://ucum.org/ucum.html and By is established name there

File: receiver/postgresqlreceiver/metadata.yaml (hunk 1 above)
Comment (dmitryax) [reply receiver/postgresqlreceiver/metadata.yaml:22]: Yes, it's defined here and there, e.g. https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/metrics/semantic_conventions#instrument-units

We here an issue to define it uniformly https://github.com/open-telemetry/opentelemetry-specification/issues/705

**Comment by bogdandrutu** [comment]
@dmitryax the unittest failure is actually a real failure, see:
```
=== RUN   TestScraper
    scraper_test.go:45: 
        	Error Trace:	scraper_test.go:45
        	Error:      	Received unexpected error:
        	            	metric Unit does not match expected: , actual: 1
        	Test:       	TestScraper
--- FAIL: TestScraper (0.08s)
=== RUN   TestScraperNoDatabaseSingle
    scraper_test.go:63: 
        	Error Trace:	scraper_test.go:63
        	Error:      	Received unexpected error:
        	            	metric Unit does not match expected: , actual: 1
        	Test:       	TestScraperNoDatabaseSingle
--- FAIL: TestScraperNoDatabaseSingle (0.08s)
=== RUN   TestScraperNoDatabaseMultiple
    scraper_test.go:81: 
        	Error Trace:	scraper_test.go:81
        	Error:      	Received unexpected error:
        	            	metric Unit does not match expected: , actual: 1
        	Test:       	TestScraperNoDatabaseMultiple
```

**Review by codeboten — APPROVED** [review]
Thanks for clarifying @dmitryax!

**Pull request merged**